### DISCUSSION TITLE
Securityonion on VM for home lab

### DISCUSSION THREAD

**Discussion by Sulaimangd** [Security-Onion-Solutions/securityonion#1878] [Unsupported Versions] [answered]
Hi,

i want to run Securityonion on a virtualization host, this is home network, and most use will be studying attacks, threats and also how to use securityonion and it's modules. i might use it also in the future to detect attacks, coming from the internet.
im going to run securityonion on Proxmox.
my questions
1- how much should be allocated to the VM in terms of memory and CPU (Internet traffic is less than 100 Mbps)
2- what is the recommended setup for securityonion for both virtualized, and physical? and for each setup what are the pros and cons (this is again for my home lab)

Thanks

**Comment by netsecninja** [comment]
I have SO 2.3.2 running on Proxmox in my home lab. I have a main NIC assigned for management and log ingestion, and a second NIC tied to my mirrored port on my home switch to sniff traffic. I have a main 100GB HD from my main SSD in Proxmox, then a 1TB SSD, which is a  dedicated physical SSD in my proxmox host just for /nsm. I found out the hard way you definitely want an SSD for log data, otherwise it seems to take forever to retrieve data. Here's the specs I used on my VM build.

![image](https://user-images.githubusercontent.com/10831043/98825500-cc808f80-23f1-11eb-9354-55f1e6103775.png)
![image](https://user-images.githubusercontent.com/10831043/98825553-db674200-23f1-11eb-9e6c-43772764a8ac.png)

**Comment by netsecninja** [comment, marked as answer]
Also, here's the recommended specs as documented. https://docs.securityonion.net/en/2.3/hardware.html#standalone-deployments
